exam/models/exam.py

```diff
@@ -271,6 +356,33 @@ class AdditionalExam(models.Model):
     weightage = fields.Char("WEIGHTAGE")
     create_date = fields.Date("Created Date", help="Exam Created Date")
     write_date = fields.Date("Updated date", help="Exam Updated Date")
+    color_name = fields.Integer("Color index of creator",
+                                compute='_compute_color_name', store=False)
+
+    @api.multi
+    def _compute_color_name(self):
+        for rec in self:
+            rec.color_name = rec.subject_id.id
```

Comment:
make sure compute functions are predefined. Place them before the field declaration.